benchmarks/instruction_counts/definitions/standard.py

```diff
@@ -0,0 +1,131 @@
+"""Default set of benchmarks."""
+
+from core.api import GroupedModules, GroupedStmts, GroupedVariants
+from core.types import FlatIntermediateDefinition
+from core.utils import flatten, parse_stmts
+from definitions.setup import Setup
+
+
+BENCHMARKS: FlatIntermediateDefinition = flatten({
+    "Empty": {
+        "no allocation": GroupedStmts(
+            r"torch.empty(())",
+            r"torch::empty({0});",
+        ),
+
+        "with allocation": GroupedStmts(
+            r"torch.empty((1,))",
+            r"torch::empty({1});",
+        ),
+
+        "overloads": GroupedVariants(
+            cpp_block=r"""
+                // <-- Setup -->
```

Comment:
Total bikeshed, but fwiw the `<--` `-->` seems a little heavy to me, if all we want to do is distinguish the label comments from ordinary ones. I'd be inclined to pitch just like `// @label` - but it's a minor point. Totally your call.

Regardless though, I agree w/Natalia that `GroupedVariants` is a little underdescribed considering how significantly it increases the expressiveness of the system. Given that it's not the job of this code to provide complete documentation, what about just 
* a line early in the README that points to this file for examples, and 
* a comment header at the top of this file that gives for constraints for each format (i.e., roughly what you have at the top of the `parse_stmts` function)

Not to do the work of an actual tutorial, just so if you're landing here, you have enough to generalize from these examples with a minimum of trial and error.